server/metrics/global_status.go

```diff
@@ -161,7 +161,10 @@ func SetMetadataOperationInContext(ctx context.Context) {
 func AddSearchBytesInContext(ctx context.Context, value int64) {
 	reqStatus, exists := RequestStatusFromContext(ctx)
 	if reqStatus != nil && exists {
-		reqStatus.writeBytes += value
+		reqStatus.searchWriteBytes += value
+	}
+	if config.DefaultConfig.GlobalStatus.DebugMessages {
+		log.Debug().Int64("search bytes written", value).Int64("Total search bytes written", reqStatus.searchWriteBytes).Msg("Added written search bytes")
```

Comment:
log field name with spaces will be hard to query. can you translate it to `search_bytes_written` and so on.